container/kubernetes/onboarding/README.md

```diff
@@ -0,0 +1,126 @@
+# IOT Agent Device OnBoarding Framework
+This Framework Blueprint for Onboarding Gateway Devices for Industry Fusion.
+
+## Installation Pre-Requisites:
+In order to setup and onboard assets with a respective gateway you need access to the following:
+
+Install and configure k3s:
+=========================
+Install the Lightweight Kubernetes k3s on gateway (root rights required).
+
+Create namespace k3s:
+==============================
+```
+kubectl create namespace oisp-devices
+```
+
+Install python3 and pip3 on your gateway:
+========================================
+```
+sudo apt install python3
+sudo apt install python3-pip
+```
+Install Packages with Pip using the Requirements Files:
+======================================================
+```
+pip3 install -r requirements.txt
+```
+GitHub repos:
+============
+
+[IndustryFusion-private-](https://github.com/mattmikulina/IndustryFusion-private-)
+
+[machine-config repo](https://github.com/mattmikulina/IndustryFusion-machine-configs)
+
+[deployment-config-repo](https://github.com/shailesh83/Gateway-Deployment)
+
+Store git credentials on cache for 30 days:
+==========================================
+```
+git config --global credential.helper cache
+git config --global credential.helper 'cache --timeout=2592000'
+git config --global user.email "you@example.com"
+git config --global user.name "Your Name"
+```
+Docker Hub:
+==========
+
+[ibn40/repositories](https://hub.docker.com/orgs/ibn40/repositories)
+
+[oisp/repositories](https://hub.docker.com/orgs/oisp/repositories)
+
+Create secret for docker hub repository with your docker credentials:
+====================================================================
+```
+kubectl -n oisp-devices create secret docker-registry regcred --docker-server=docker.io --docker-username=YOUR_DOCKER_USERNAME --docker-password=YOUR_DOCKER_PASSWORD --docker-email=YOUR@EMAIL.COM
+```
+
+OISP Platform:
+=============
+Get access to OISP Platform Dashboard.
+[platform-industry-fusion](https://platform.industry-fusion.com)
```

Comment:
please remove platform url and add latest.oisp.info

Reply:
done